src/content/staking/withdrawals/index.md

```diff
@@ -0,0 +1,170 @@
+
+### How do I prepare? {#how-do-i-prepare}
+
+<WithdrawalsTabComparison />
+
+### Important notices {#important-notices}
+
+Providing a withdrawal address is a required step for any validator account before it will be eligible to have ETH withdrawn from its balance.
+
+<InfoBanner emoji="⚠️" isWarning>
+  <strong>Each validator account can only be assigned a single withdrawal address, one time.</strong> Once an address is chosen and submitted to the Beacon Chain, this cannot be undone or changed again. Double-check ownership and accuracy of the address provided before submitting.
+</InfoBanner>
+
+Failure to update withdrawal credentials will simply leave the ETH locked in the validator account as it has been until a withdrawal address is provided. There is <strong>no threat to your funds in the meantime</strong> for not providing this—only the loss of opportunity.
+
+## Exiting staking entirely {#exiting-staking-entirely}
+
+Providing a withdrawal address is required before _any_ funds can be transferred out of a validator account balance.
+
+Users looking to exit staking entirely and withdrawal their full balance back must also sign and broadcast a "voluntary exit" message with validator keys which will start the process of exiting from staking.
+
+The process of a validator exiting from staking takes variable amounts of time, depending how many others are exiting at the same time. Once complete, this account will no longer be responsible for performing validator network duties, is no longer eligible for rewards, and no longer has their ETH "at stake". At this time the account with be marked as fully “withdrawable”.
+
+Once an account is flagged as "withdrawable", and withdrawal credentials have been provided, there is nothing more a user needs to do aside from wait. Accounts are automatically and continuously swept for eligible exits, and your account balance will be transferred in full during the next sweep (also known as a "full withdrawal").
+
+## When are staking withdrawals enabled? {#when}
+
+Withdrawals will be enabled through a two-part simultaneous network upgrade, **Shanghai + Capella**.
+
+<ShanghaiCapella />
+
+## How do withdrawal payments work? {#how-do-withdrawals-work}
+
+Once a validator account has a withdrawal address registered, rewards payments for eligible ETH will happen automatically.
+
+Instead of requiring stakers to manually submit a transaction requesting a particular amount of ETH to be withdrawn, withdraws are designed to automatically transfer out any amount of ETH that is not actively at stake—**no gas required**.
+
+### Validator "sweeping"
+
+With each block, the proposing validator must include a list of withdrawals to process. Each validator is evaluated for possible withdrawals in order, evaluated as follows:
+
+| Decision                                     | Yes                                   | No                                                     |
```

Comment:
Nice table! I think this could be even better as a flowchart. Perhaps a future iteration with some design help? :)

Reply:
Yeah definitely! Would love to have it as part of THIS iteration, but as with any infographic/flow chart type things, they're easy to make in English-only non-responsive PNG's, but will take a little dev work if we want it to be intl/mobile friendly. TBD on this one, but design suggestions certainly welcome ❤️ cc: @konopkja